tests/test_cli.py

```diff
@@ -425,17 +443,13 @@ def test_run_empty_file(self):
             cli.run(('-f', 'parsable', path))
         self.assertEqual((ctx.returncode, ctx.stdout, ctx.stderr), (0, '', ''))
 
+    @unittest.skipIf(not utf8_paths_supported(), 'UTF-8 paths not supported')
     def test_run_non_ascii_file(self):
-        path = os.path.join(self.wd, 'non-ascii', 'éçäγλνπ¥', 'utf-8')
-
-        # Make sure the default localization conditions on this "system"
-        # support UTF-8 encoding.
-        try:
-            locale.setlocale(locale.LC_ALL, (None, 'UTF-8'))
-        except locale.Error:
-            self.skipTest('no UTF-8 locale available')
-        self.addCleanup(locale.setlocale, locale.LC_ALL, (None, None))
+        if sys.version_info < (3, 7):
```

Comment:
added a check for python version here - because we basically identified through the previous test results, that we only need to manually set the locale to UTF-8 to support those paths for those earlier versions.